changelog

```diff
@@ -1,3 +1,24 @@
+turnkey-gnusocial-17.0 (1) turnkey; urgency=low
+
+  * Updating gnusocial and its dependencies to tkldev v17.0.
+
+  * Updated libinithooks dialog_wrapper to refactored version 17.
+
+  * Updated gnusocial repository tarball link from gnu.io to notabug.com.
+
+  * Removed code error in conf.d/main responsible for renaming directory /var/www/gnusocial.
+
+  * Updated moved index.php file for curl command to complete apache install in gnusocial.conf file.
+  
+  * Update new directory locations needed for apache gnu install with curl command.
+
+  * Installed php-bcmath package to complete updated apache gnu install thorugh curl command. 
+
+  * Note: Please refer to turnkey-core's 16.1 changelog for changes common to all
```

Comment:
Should be:

> [...] refer to v17.0 changelog [...]

Reply:
I went through and changed this for you as well.